### PR TITLE
Validate Apify proxy country

### PR DESCRIPTION
**Pull request by fnesveda**
Adds validation for the selected country for Apify proxy.

### CODE REVIEW

File: src/utilities.client.js [hunk 1]
```diff
@@ -399,6 +400,11 @@ function validateProxyField(fieldKey, value, isRequired = false, options = null)
     // If Apify proxy is not used skip additional checks
     if (!useApifyProxy) return fieldErrors;
 
+    // If Apify proxy is used, check if there is a selected country and if so, check that it's valid (empty or a valid country code)
+    if (apifyProxyCountry && apifyProxyCountry !== '' && !countries[apifyProxyCountry]) {
```
Comment (jancurn) [inline src/utilities.client.js]: The `apifyProxyCountry !== ''` condition is redundant, the first one will cover this.

Comment (fnesveda) [reply src/utilities.client.js]: Fixed in fdb4a6c.

**Comment by fnesveda** [comment]
@gippy That's a good point, added the check in 8c54d99.

File: src/intl.js [hunk 2]
```diff
@@ -22,6 +22,10 @@ const intlStrings = {
         'You currently do not have access to proxy groups: {groups}',
     'inputSchema.validation.customProxyInvalid':
         'Proxy URL "{invalidUrl}" has invalid format, it must be http://[username[:password]]@hostname:port.',
+    'inputSchema.validation.apifyProxyCountryInvalid':
+        'Country code "{invalidCountry}" is invalid.',
```
Comment (mtrunkat) [inline src/intl.js]: Let's metion in error that `ISO 3166-1 alpha-2 codes` are supported  https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2

**Pull request merged**